### DISCUSSION TITLE
posts/2021-07-15-publishing-a-unity-webgl-game-from-scratch-in-under-30-minutes/

### DISCUSSION THREAD

**Discussion by giscus** [Razenpok/razenpok.github.io#3] [Announcements]
# posts/2021-07-15-publishing-a-unity-webgl-game-from-scratch-in-under-30-minutes/

Messing with code instead of making releases

https://refactoring.ninja/posts/2021-07-15-publishing-a-unity-webgl-game-from-scratch-in-under-30-minutes/

**Comment by DanielBrittainJr** [comment]
Great tutorial. I was able to get it working!

This is awesome!

**Comment by arthurmougin** [comment]
Hi, this article is now out of date. There is now an error in the activation action :
> Error: This request has been automatically failed because it uses a deprecated version of `actions/upload-artifact: v2`. Learn more: https://github.blog/changelog/2024-02-13-deprecation-notice-v1-and-v2-of-the-artifact-actions/

**Comment by razenpok** [comment]
Hi @arthurmougin 

There was a change in the Unity activation process, and a separate GitHub action is no longer needed
You should remove it and follow the updated steps from here: https://game.ci/docs/github/activation/
